### DISCUSSION TITLE
How to Configure Pixi to Download from Mirror Sites

### DISCUSSION THREAD

**Discussion by harnvo** [prefix-dev/pixi#1696] [Q&A] [answered]
Thanks for your great work.

I am currently in a region where downloading dependencies from default sites are unreasonably slow. For other dependency managers (e.g. apt, pip, conda) I would usually configure them to download dependencies from mirror sites close to my region. However, I cannot find any documentation about how to do the same for pixi.

Is this feature supported by pixi yet? If yes, how? And if not, is there anything else I can do to resolve this issue?

**Comment by pavelzw** [comment, marked as answer]
https://pixi.sh/latest/reference/pixi_configuration/#mirror-configuration
